x/stakeibc/client/cli/tx_register_host_zone.go

```diff
@@ -34,6 +40,31 @@ func CmdRegisterHostZone() *cobra.Command {
 			if err != nil {
 				return err
 			}
+
+			minRedemptionRateStr, err := cmd.Flags().GetString(FlagMinRedemptionRate)
```

Comment:
Does [this](https://github.com/Stride-Labs/stride/pull/606/files#diff-876e81fc916d4f89abe90087cb7ea268493301ab0b6798ec2156090ecde7f5b0R44-R67) error throwing force setting a min and max RR? Don't we want to allow option to leave it default?

- The [logic that checks safety bounds](https://github.com/Stride-Labs/stride/compare/772fed33504795a59a1e2b776dcae24ae7072a2a...a0617ca#diff-9f00fc01cf07011bf0edafa3a5f799d0fc4092028931d9972150358253c63d2cR267-R277) implies we *can* have default min/max rates 
- this logic also seems to imply you could register a hz without a custom min and max

Reply:
I believe the error caught here is if `GetString` fails, which should only happen if they pass something invalid for that arg. 

If no flag is set, `FlagMinRedemptionRate` should be `""` and `err` should be `nil`.
Then, minRedemptionRate gets set to the Zero and in msg_server, we catch that it's zero and override with the default